test/BaselineOutput/SingleRelease/LightGBMBinary/LightGBM-TrainTest-breast-cancer-out.txt

```diff
@@ -0,0 +1,45 @@
+maml.exe TrainTest test=%Data% tr=LightGBMBinary{nt=1 nl=5 mil=5 lr=0.25 iter=20 mb=255} cache=- dout=%Output% loader=Text{sparse- col=Attr:TX:6 col=Label:0 col=Features:1-5,6,7-9} data=%Data% out=%Output% seed=1
```

Comment:
The `SingleRelease` baseline files appears duplicated with the `SingleDebug` baseline files.  Are both sets of files required?  or can they be consolidated into a one set of baselines?

Reply:
Hi Eric, the outputs differ at times. For example if the native code changes it can differ and that is when we want to understand why the output changed. For now we should follow the practice and maintain two outputs and if this is something that we want to optimize then it would be a separate issue.

---
In reply to: [197830739](https://github.com/dotnet/machinelearning/pull/392#discussion_r197830739) [](ancestors = 197830739)

Reply:
[https://github.com/dotnet/machinelearning/issues/410](https://github.com/dotnet/machinelearning/issues/410) just in case.

---
In reply to: [197978265](https://github.com/dotnet/machinelearning/pull/392#discussion_r197978265) [](ancestors = 197978265,197830739)